app/views/submissions/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= simple_form_for @submission  do |form| %>
+  <% if @submission.errors %>
```

Comment:
I believe this will always yield `true`, as an AR object will always have `errors`. So, you might want a `present?` check.